_posts/2019-02-08-resize-images-on-the-fly.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "How to resize your images on-the-fly with OpenFaaS"
+description: In this post Tarun from IconScout walks us through how his company is leveraging OpenFaaS with Node.js and OpenFaaS to boost image resizing and conversion speeds of up to 4-5x.
+date: 2019-02-04
+image: https://images.pexels.com/photos/440731/pexels-photo-440731.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260 # temporary
+categories:
+  - end-user
+  - node
+  - tutorial
+author_staff_member: alex
+dark_background: false
+---
+
+I’d like to introduce you to Tarun who is our guest-writer for today’s end-user blog post.
+
+> Author bio: [Tarun Mangukiya](https://twitter.com/tarunmangukiya?lang=en-gb) is co-founder and Product Manager at [IconScout](https://iconscout.com). He’s working on Frontend, Docker, and Serverless and many more. They are working to migrate their codebase to serverless over the time. Currently, their all the processing jobs are powered by OpenFaaS.
+
+<img src="/images/resize-on-fly/tarun.jpeg" width="25%" height="25%" />
+
+![](https://docs.openfaas.com/images/logos/iconscout.svg)
+
+At [IconScout](https://iconscout.com) over the last two years we’ve been building a platform to solve different types of image resizing and processing problems for our customers. We’re on a mission to bring design assets on hand through the next-generation Design Resource Marketplace and Design Asset Management tools.
+
+Read more: [About IconScout](https://iconscout.com/about)
+
+Our v1 codebase was developed using PHP and ImageMagick, which served us well at the time. We used VMs to resize images as a batch process and then store them, but in May last year we started to hit the limits of our original approach and found out about OpenFaaS.
```

Comment:
What were the limits, how did OpenFaaS alleviate these?

Reply:
For limits I think we've a separate blog that I've linked at the bottom.